app/views/search/_filters.html.slim

```diff
@@ -25,13 +25,9 @@
         | Tags
         span.filters__clear= link_to 'Clear', root_path, class: 'gray-light pull-right'
     .tags-list
-      = link_to '#urban sustainability (245)', root_path, class: 'tags-list__tag'
-      = link_to '#solar energy (148)', root_path, class: 'tags-list__tag'
-      = link_to '#wind (136)', root_path, class: 'tags-list__tag tags-list__tag--selected'
-      = link_to '#ocean (136)', root_path, class: 'tags-list__tag tags-list__tag--selected'
-      = link_to '#solar energy (148)', root_path, class: 'tags-list__tag'
-      = link_to '#solar (148)', root_path, class: 'tags-list__tag'
-      .clearfix
+      - ActsAsTaggableOn::Tag.most_used(10).each do |tag|
```

Comment:
Can we move `ActsAsTaggableOn::Tag.most_used(10)` to the controller. 
Reason: Any complex (even simple hehe) logic should be on the view.

Reply:
@nsanta -- Thibault is no longer working on this project. Are you comfortable making the change you requested in a separate PR?